### ISSUE TITLE
Register CT and MR (T1 and T2) within same subject

### ISSUE THREAD

**Issue by bhatrana** [voxelmorph/voxelmorph#446]
Register CT and MR (T1 and T2) within same subject

I am trying to register CT (fixed) , MR moving (T1 aswell as T2). 

I used more than 150 training sets for this experiements, The used loss function is global MI . 

I found moved MR as scaled. 

Attached is the screen shots. 

What could be the issues and how can i solve to register CT and MR ?

![image](https://user-images.githubusercontent.com/79718549/174106112-72be93d3-aad0-4e60-8a9c-08ff53327ac8.png)

**Comment by ly1998117** [comment]
Try SynthMorph?
